index.html

```diff
@@ -0,0 +1,215 @@
+                            <ul class="user-links desktop">
+                                <li>English</li>
+                                <li>MyPage</li>
+                                <li>Logout</li>
+                            </ul>
+                        </div>
+                        <div class="menu">
+
+                            <img
+                                src="/first-capstone-project/assets/icons/nav-icon.png"
+                                class="mobile"
+                                id="menu" onclick="menuOverlay()" />
+                            <h2 id="logo"><a href="index.html">Amanna Trust</a>
+                            </h2>
+
+                            <img
+                                src="/first-capstone-project/assets/icons/close-icon.png"
+                                class="mobile"
+                                id="menu-close" />
+
+                            <ul class="nav-links" id="nav-links">
+
+                                <li class="nav-link active"><a
+                                        href="/first-capstone-project/index.html">Home</a></li>
+                                <li class="nav-link"><a
+                                        href="/first-capstone-project/pages/about.html">About</a></li>
+                                <li class="nav-link">Program</li>
+                                <li class="nav-link">join</li>
+                                <li class="nav-link">Sponsor</li>
+                                <li class="nav-link">News</li>
+                            </ul>
+                            <div class="campaign-donate mobile desktop">
+                                <p class="campaign-donate-text">Donate</p>
+                            </div>
+                            <div class="close">
+                                <img src="assets/icons/close-icon.png"
+                                    class="mobile"
+                                    id="menu-close" />
+                            </div>
+                        </div>
+
+                    </div>
+
+                </nav>
+            </header>
+            <section class="hero">
+                <div class="headline">
+                    <p>Your help goes a long way!</p>
+                    <h1 class="title-h1">Help us reach needy families and
+                        empower them get
+                        access to education. </h1>
+                </div>
+
+                <div class="card">
+                    <p class="headline-text">We aim to shape possibilities of
+                        under privileged
+                        children by providing a stepping stone to acquiring
+                        education and better living conditions.</p>
+                </div>
+
+                <q>The greatness of humanity is not in being human, but in being
+                    humane. – Mahatma Gandhi
+                </q>
+
+            </section>
```

Comment:
great work was done here as well , but there still are some small issues to fix please consider increasing the text size of the first headline and the big title as well as the margin between all this sections components this will help also increase the space covered by this view as you see in the design it almost covers the view hight 

![Capture d’écran du 2023-08-02 00-04-49](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/e567915f-7941-4f08-8a53-0d95839cccd8)
![Capture d’écran du 2023-08-02 00-04-24](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/92cf2375-a23d-4732-b386-651d587a1576)